packages/shortest/src/types/browser.ts

```diff
@@ -27,7 +27,9 @@ export type BrowserAction =
   | "type"
   | "key"
   | "run_callback"
-  | "navigate";
+  | "navigate"
+  | "sleep"
+  | "check_email";
```

Comment:
Adding these types to the list of our Browser actions. This will help us to validate the AI will not send an action that we have not implemented yet.